packages/spec/src/index.ts

```diff
@@ -1,6 +1,106 @@
-import type * as v1 from "./v1.ts";
+export declare namespace v1 {
+  /**
+   * The Standard Schema interface.
+   */
+  export interface StandardSchema<Input = unknown, Output = Input> {
+    /**
+     * The Standard Schema properties.
+     */
+    readonly "~standard": StandardSchemaProps<Input, Output>;
+  }
 
-export type StandardSchema = v1.StandardSchema;
+  /**
+   * The Standard Schema properties interface.
+   */
+  export interface StandardSchemaProps<Input = unknown, Output = Input> {
+    /**
+     * The version number of the standard.
+     */
+    readonly version: number;
+    /**
+     * The vendor name of the schema library.
+     */
+    readonly vendor: string;
+    /**
+     * Validates unknown input values.
+     */
+    readonly validate: (
+      value: unknown,
+    ) => StandardOutput<Output> | Promise<StandardOutput<Output>>;
+    /**
+     * Inferred types associated with the schema.
+     */
+    readonly types?: (input: Input) => Output;
```

Comment:
@ssalbdivad suggested alternative names for this property, such as `infer`, `inference` or `inferred`. Feel free to discuss this here. I think I would stick with `types`, but I am not too opinionated about it.

Reply:
Probably pretty low risk since I've never heard of anyone doing this with arktype's `infer` prop, but I'm totally fine with `.inferred` here.

Reply:
Hmm... I don't like `inferred`. 🙈 @ssalbdivad are you ok with keeping `types`?